pandas/core/strings.py

```diff
@@ -318,6 +322,48 @@ def str_contains(arr, pat, case=True, flags=0, na=np.nan, regex=True):
     --------
     match : analogous, but stricter, relying on re.match instead of re.search
 
+    Examples
+    --------
+    >>> s = pd.Series(['Mouse', 'dog', 'house and parrot', '23', np.NaN])
+    >>> s
+    0               Mouse
+    1                 dog
+    2    house and parrot
+    3                  23
+    4                 NaN
+    dtype: object
+
+    >>> s.str.contains('og', regex=False)
+    0    False
+    1     True
+    2    False
+    3    False
+    4      NaN
+    dtype: object
+
+    >>> s.str.contains('og', na=False, regex=True)
```

Comment:
These examples are good but some text to direct the user on what to look at preceding each would be helpful. For example here I would prepend something like:

```With the above examples, you'll note that NA values in the caller return `np.nan` by default and as a result cast the dtype of the returned object to `object`. You can control this behavior by specifying a an alternate `na` argument```